allergies/allergies.py

```diff
@@ -13,30 +13,18 @@
 class Allergies(object):
 
     def __init__(self, score):
-        self.score = score
-        self.allergen_list = self.lst
+        self.allergen_list = []
+        self.score = score % 256
```

Comment:
That's really cool! That makes the `lst` generation way faster 👍In real codebase it might be good to explain that either with a better name of the variable (though I don't see anything good here really) or/and with a comment above that line and/or with docstrings for `__init__` (that is - a commentary of the method usage).